src/business-logic/services/ingest-data-changed-service.ts

```diff
@@ -66,7 +83,8 @@ export class IngestDataChangedService implements DataChangeService {
   private readonly logger: Logger
   private isExecutingEvent: boolean = false
   private lastBulkExecutionStartTimestamp: number = 0
-  private readonly rundownIdsToBuild: Set<string> = new Set<string>()
+  private readonly rundownIdsToBuild: Set<string> = new Set()
```

Comment:
How does one build a rundown id?
If it is the timeline, then the same syntax as `rundownIdsToGenerateActionsFor` could be used.